book/src/setup.md

```diff
@@ -124,15 +123,24 @@ and applying power again.
 
 ### DFU Upload
 
-If an SWD/JTAG probe is not available,
-you can flash firmware using only a micro USB cable
-plugged in to the front of Stabilizer, a DFU utility, and a jumper to activate
-the bootloader.
+If an SWD/JTAG probe is not available, you can flash firmware using only a micro USB cable plugged
+in to the front of Stabilizer, and a DFU utility.
+
+> **Note:** If there is already newer firmware running on Stabilizer that supports the USB serial
+> interface, there is no need to remove Stabilizer from the crate or disconnect any existing
+> connectors/power supplies or to jumper the BOOT0 pin. Instead, open the serial port on Stabilizer
+> and request it to enter DFU mode:
+> ```bash
+> python -m serial <serial-port>
+> > platform dfu
+> ```
+>
+> After the device is in DFU mode, use the `dfu-util` command specified in the instructions below,
```

Comment:
The instruction flow here could be simplified and clarified a lot. But let's leave it.